src/input/datetime.rs

```diff
@@ -264,8 +265,17 @@ pub fn bytes_as_date<'a>(input: &'a impl Input<'a>, bytes: &[u8]) -> ValResult<'
     }
 }
 
-pub fn bytes_as_time<'a>(input: &'a impl Input<'a>, bytes: &[u8]) -> ValResult<'a, EitherTime<'a>> {
-    match Time::parse_bytes(bytes) {
+pub fn bytes_as_time<'a>(
+    input: &'a impl Input<'a>,
+    bytes: &[u8],
+    microseconds_overflow_behavior: MicrosecondsPrecisionOverflowBehavior,
+) -> ValResult<'a, EitherTime<'a>> {
+    match Time::parse_bytes_with_config(
+        bytes,
+        TimeConfig {
+            microseconds_precision_overflow_behavior: microseconds_overflow_behavior,
```

Comment:
Well speak of the devil, if this is the only field currently, I think it's fine to just use the line above, (unless, as noted, this API really is public in some way).